app/src/main/java/com/infomaniak/drive/ui/fileList/SelectFolderActivity.kt

```diff
@@ -108,7 +116,7 @@ class SelectFolderActivity : BaseActivity() {
     }
 
     private fun MutableList<Int>.addNavigationIdsRecursively(folderId: Int, userDrive: UserDrive) {
-        FileController.getParentFileProxy(folderId, mainViewModel.realm)?.id?.let { parentId ->
+        FileController.getParentFile(folderId, userDrive, getRealm(userDrive))?.id?.let { parentId ->
```

Comment:
See with @sirambd  if we can use the getParentFile instead of the proxy here.

Reply:
Yes, it's a bit cumbersome to make a whole copy just to get an id, you have to go through a proxy.